src/components/board-grid.tsx

```diff
@@ -16,22 +22,95 @@ export interface BoardProps {
 
 export const BoardGrid: FC<BoardProps> = ({ game }) => {
   const { gameId } = useParams();
-  const [theGame, isLoading] = useRunningGame(gameId!);
+  const [theGame] = useRunningGame(gameId!);
 
   if (!theGame) {
     return <p>"Loading"</p>;
   }
 
-  const { players } = theGame;
-  const [{ name: player1Name }] = players;
+  const { players, turn } = theGame;
+  const [{ name: player1Name }, { name: player2Name }] = players;
 
-  console.log(player1Name);
+  if (theGame.state !== "GAME_OVER") {
+    return (
```

Comment:
This is getting quite hard to read. It would be nice to split each of these returns out into their own components representing each game state.